patacrep/content/__init__.py

```diff
@@ -157,23 +219,23 @@ def render(context, content):
         rendered += elem.render(context) + EOL
         previous = elem
 
-    if isinstance(last, Content):
+    if isinstance(last, ContentItem):
```

Comment:
Cette condition pourrait être remplacée par `if last:`, puisque les éléments qui ne sont pas des `ContentItem` sont déjà omis dans la boucle `for`